python/cutlass/library_defaults.py

```diff
@@ -52,19 +53,23 @@
 # Strip any additional information from the CUDA version
 _cuda_version = __version__.split("rc")[0]
 
-# Check that Python CUDA version exceeds NVCC version
-_nvcc_version = cutlass.nvcc_version()
-_cuda_list = _cuda_version.split('.')
-_nvcc_list = _nvcc_version.split('.')
-for val_cuda, val_nvcc in zip(_cuda_list, _nvcc_list):
-    if int(val_cuda) < int(val_nvcc):
-        raise Exception(f"Python CUDA version of {_cuda_version} must be greater than or equal to NVCC version of {_nvcc_version}")
-
-if len(_nvcc_list) > len(_cuda_list):
-    if len(_nvcc_list) != len(_cuda_list) + 1:
-        raise Exception(f"Malformatted NVCC version of {_nvcc_version}")
-    if _nvcc_list[:-1] == _cuda_list and int(_nvcc_list[-1]) != 0:
-        raise Exception(f"Python CUDA version of {_cuda_version} must be greater than or equal to NVCC version of {_nvcc_version}")
+if not os.getenv("CUTLASS_USE_SYCL"):
```

Comment:
```suggestion
if not os.getenv("CUTLASS_USE_SYCL"):
```
I see `CUTLASS_USE_SYCL` is also used in `__init__.py`, but I wonder if we should align that with `CUTLASS_ENABLE_SYCL`, the preprocessor symbol used in the c++ src?

Reply:
`CUTLASS_USE_SYCL` is only a temporary solution to switch between different backends in the Python interface. The plan is to replace that with something that allows to switch backends directly in Python. 

As it is only temporary, I wanted to keep it separate from the existing symbols.